config/checkstyle_resources_suppressions.xml

```diff
@@ -74,4 +74,78 @@
   <suppress checks="RegexpSingleline"
     files="[\\/]InputJavadocTagContinuationIndentationGuavaFalsePositive\.java"/>
 
+  <!-- intentional problem for testing -->
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputJavadocWithCrAsNewline\.javadoc"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputLeadingAsterisksExtended\.javadoc"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputListWithUnclosedItemInUnclosedParagraph\.javadoc"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputMixedCaseOfHtmlTags\.javadoc"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputUnclosedParagraphFollowedByJavadocTag\.javadoc"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputFileTabCharacter\.java"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputMainGenerateXpathSuppressionsTabWidth\.java"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputXpathQueryGeneratorTabWidth\.java"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputFileTabCharacterSimple\.java"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputIndentationUseTabs\.java"/>
+  <suppress checks="FileTabCharacter"
+    files="[\\/]InputCheckerTabCharacter\.txt"/>
+  <suppress checks="FileTabCharacter"
+    files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]indentation[\\/]indentation[\\/]Input.*\.java"/>
+  <suppress checks="FileTabCharacter"
+    files="src[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]filters[\\/]suppresswithplaintextcommentfilter[\\/]Input.*"/>
```

Comment:
Please trim path to just `[\\/]test[\\/]resources[\\/]` and checkname (indentation, suppresswithplaintext...) with the input on the end.

Reply:
That will suppress more files than what is necessary. Moreover, these will be fixed in subsequent PRs if possible. Please refer PR description.